update-orphaning/index-old.html

```diff
@@ -0,0 +1,513 @@
+<!DOCTYPE html>
```

Comment:
Before deciding to copy it I discussed this with ddurst and decided that it should be kept for awhile since it has data going back further than this dashboard. We'll evaluate removing it sometime in the future.